.github/workflows/build_and_test.yml

```diff
@@ -254,7 +254,7 @@ jobs:
     - name: Install pybind11
       run: conan create --profile=tket recipes/pybind11
     - name: Build pytket (3.8)
-      if: github.event_name == 'push'
+      if: github.event_name == 'pull_request'
```

Comment:
Presumably this change (and below) should be reverted.

Reply:
Yes